cloud/azure/bin/lib/azure_log.sh

```diff
@@ -17,6 +17,10 @@ readonly AZURE_LOG_FILE_NAME="civiform_deployment_log.txt"
 #   LOG_TEMPFILE: the path to a tempfile containing the deploy log.
 #######################################
 function azure_log::fetch_log_file() {
+  if [[ "${CIVIFORM_MODE}" == "test" ]]; then
```

Comment:
suggestion (blocking): extract a function
```suggestion
  if civiform_mode::is_test; then
```